### PR TITLE
Update CMakeLists.txt file to avoid compilation issue.

### PR DESCRIPTION
**Pull request by evielm9awda**
Issue:
Screenshot_2024-07-11-11-00-28_primary
Critical issues:
- Modified the `write` call in the `handlefd` function to only write the actual number of peers, preventing potential buffer overflows when sending peer lists
- Added error checking for `write` operations in the `handlefd` function. This enhancement provides better error reporting and allows for more robust error handling.
- Added a NULL check before accessing the first character of the `fgets` result in the client function. This prevents potential crashes or undefined behavior when reading user input.

### CODE REVIEW

File: CMakeLists.txt
Comment (Sharp0802) [inline CMakeLists.txt:1]: Sorry, subdirectory name should be `vnsl`, not `vcl`.
Please change directory name, not content of `CMakeLists.txt`.

**Review by Sharp0802 — CHANGES_REQUESTED** [review]
Sorry, subdirectory name should be `vnsl`, not `vcl`.
Please change directory name, not content of `CMakeLists.txt`.

**Comment by evielm9awda** [comment]
> Sorry, subdirectory name should be `vnsl`, not `vcl`. Please change directory name, not content of `CMakeLists.txt`.

Ok, gotcha

**Review by Sharp0802 — APPROVED** [review]
LGTM :+1:

**Pull request merged**